docs/developer-docs/build/frontend/webpack-config.md

```diff
@@ -4,15 +4,15 @@ the IC allows you to host Web 3.0 frontends for your dapps, using our [JavaScrip
 
 Here are some quick links to tutorials with example code for various stages of developing your frontend dapp:
 
--   A tutorial on building a React dapp [Customize the frontend](custom-frontend)
+-   A tutorial on building a React dapp [Customize the frontend](../custom-frontend)
```

Comment:
```suggestion
-   A tutorial on building a React dapp [Customize the frontend](../custom-frontend.md)
```
all links should contain file extensions. Otherwise we get problems with 404s